aten/src/ATen/native/cuda/CUDAUnaryOps.cpp

```diff
@@ -39,7 +40,11 @@ Tensor& _clamp_min_out_cuda(Tensor& result, const Tensor& self, Scalar min) {
   return _th_clamp_min_out(result, self, min);
 }
 
-// These are just forwarding stubs
+Tensor& _fill__cuda(Tensor& self, Scalar value) {
+  return at::legacy::th::_th_fill_(self, value);
```

Comment:
Just implement the CUDA version using `gpu_unary_kernel`. You'll need to rename this file to have the `.cu` suffix. It should be `UnaryOpsKernel.cu` to match `BinaryOpsKernel.cu`.

This will simplify dispatch and bring down the number of `_fill_` functions.

Reply:
Will do this in a separate patch.